android/orgaincmaps/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Why are you creating a new application? organicmaps subfolder is not needed.

Reply:
We have to create a new module for Android Automotive Os support 
https://developer.android.com/training/cars/apps/automotive-os

Reply:
Very few car models right now support Android OS, I guess Android Auto is more popular